src/home/components/Notification/Notification.tsx

```diff
@@ -12,24 +14,23 @@ import {
   StyledNotificationContainer,
 } from './Notification.style';
 
-const Dummy = [
-  '그거 아시나요?? 지금은 2월이라는 거? 당연함 2월임',
-  '안녕하세요~~ 공지입니다.',
-  '가나다라마바사아자차카타파하갸냐댜랴먀뱌샤야쟈챠캬탸퍄햐거너더러머버서어저처커터퍼허겨녀뎌려며벼셔여져쳐켜텨펴혀',
-  '이거슨 테스트여',
-];
-
-// TODO: 추후 공지사항 API 연결 확인 필요
-export const Notification = () => {
+const NotificationContent = () => {
+  const { data } = useGetAnnouncement();
   const [currentIndex, setCurrentIndex] = useState(1);
   const [activeTransition, setActiveTransition] = useState(true);
   const slideRef = useRef<HTMLDivElement>(null);
-  const notificationArray = [Dummy[Dummy.length - 1], ...Dummy, Dummy[0]].map(
-    (notification, index) => ({
-      id: index + 1,
-      notification,
-    })
-  );
+
+  const announcements = data?.announcementList ?? [];
+
+  const notificationArray =
+    announcements.length > 0
+      ? [announcements[announcements.length - 1], ...announcements, announcements[0]].map(
+          (announcement, index) => ({
+            id: index + 1,
+            notification: announcement.title,
+          })
+        )
+      : [];
```

Comment:
아래 무한 스크롤 로직에서는 역방향 슬라이딩을 고려하여 배열의 마지막 요소를 배열의 처음에 추가하는 것 같은데 현재 컴포넌트에서는 정방향 슬라이딩만 고려하면 될 것 같아 자연스러운 슬라이딩 효과를 위해 배열의 첫번째 요소만 배열의 마지막 위치에 추가해도 될 것 같네요!

```suggestion
  const notificationArray = [...announcements, announcements[0]].map((announcement, index) => ({
    id: index + 1,
    notification: announcement.title,
  }));
```

Reply:
적극 반영!!!
```
 if (currentIndex === notificationArray.length - 1) {
    InfiniteSlideHandler(0);
  }
```
이 부분도 바뀐 배열 추가 방법에 따라 수정하였습니다!

[refactor: 배열 추가 방법을 수정하여 무한 슬라이드 로직 단순화](https://github.com/yourssu/Soomsil-Web/pull/205/commits/a75b4e43a73cb21df2191e51330b802462b3f78f)